src/v2/Components/BankDebitForm/BankDebitForm.tsx

```diff
@@ -0,0 +1,60 @@
+import { FC, useState } from "react"
+import { PaymentElement, useStripe, useElements } from "@stripe/react-stripe-js"
+import { Box, Button, Spacer } from "@artsy/palette"
+import { useSystemContext } from "v2/System"
+import { LoadingArea } from "../LoadingArea"
+
+export const BankDebitForm: FC = () => {
+  const stripe = useStripe()
+  const elements = useElements()
+  const { user } = useSystemContext()
+  const [isPaymentElementLoading, setIsPaymentElementLoading] = useState(true)
+
+  const handleSubmit = async event => {
+    event.preventDefault()
+
+    if (!stripe || !elements || !user) {
+      return
+    }
+
+    const { error } = await stripe.confirmSetup({
```

Comment:
I'm a bit confused by `stripe.confirmSetup` vs. ACH-specific `stripe.collectBankAccountForSetup` and `stripe.confirmUsBankAccountSetup`. Will also do some more research to get a better understanding.

Reply:
It is my understanding that `stripe.confirmUsBankAccountSetup` is done during the Payment Element account selection process. The stripe docs for implementing ACH do not include using the Payment Element, so I had to go back and forth between the ACH-specific doc and the Payment Element implementation doc to get this flow.

Reply:
but yes I'm also a bit confused by this as well

Reply:
Still curious the difference here between using `stripe.confirmSetup` and `stripe.confirmUsBankAccountSetup`. Definitely non-blocking for this PR, and we can test in the followup.